scripts/pynag

```diff
@@ -690,6 +700,45 @@ def pretty_print(objects, attributes=None):
         pre = "-"*10
         post = "-"*(80-10-len(message))
         print pre + message + post
+
+def get_nagios_attributes(obj):
+    obj_attributes = {}
+    for attribute, value in obj.items():
+        # Don't return pynag internal attributes
+        # except the object type
+        if attribute == "id":
+            continue
+        if attribute == "meta":
+            obj_attributes["object_type"] = value['object_type']
```

Comment:
If object_type is already leaked out (I agree, we kind of need it). Is there any reason to remove meta and id ?